instrumentation/runtime/runtime.go

```diff
@@ -120,7 +118,7 @@ func Start(opts ...Option) error {
 
 func (r *runtime) register() error {
 	startTime := time.Now()
-	uptime, err := r.meter.AsyncInt64().UpDownCounter(
+	uptime, err := r.meter.Int64ObservableUpDownCounter(
```

Comment:
I don't think it's worth opening a separate PR, but I noticed this should be just an async counter since it's monotonic. Potentially worth changing as part of switching to new API:
```suggestion
	uptime, err := r.meter.Int64ObservableCounter(
```

Reply:
I would suggest fixing this in a follow-up PR.